readme.md

```diff
@@ -1,15 +1,3 @@
-A conversion of the OpenAI OpenAPI to TypeSpec.
+# OpenAPI spec for the OpenAI API
 
-There are some deltas:
-
-### Changes to API Semantics:
-
-- Many things are missing defaults (mostly due to bug where we can't set null defaults)
-- Error responses have been added.
-- Where known, the `object` property's type is narrowed from string to the constant value it will always be
-
-### Changes to API metadata or OpenAPI format
-
-- Much of the x-oaiMeta entries have not been added.
-- In some cases, new schemas needed to be defined in order to be defined in TypeSpec (e.g. because the constraints could not be added to a model property with a heterogeneous type)
-- There is presently no way to set `title`
+This repository contains an [OpenAPI](https://www.openapis.org/) specification for the [OpenAI API](https://platform.openai.com/docs/api-reference).
```

Comment:
it has both readme.md and README.md, windows are not casesensitive, may need to revert it back.